lib/pfab/cli.rb

```diff
@@ -309,6 +309,8 @@ def get_current_sha
     def set_kube_context
       str = "kubectl config use-context #{config["envs"][$env.to_s]["context"]}"
       puts_and_system str
+      str = "kubectl config set-context --current --namespace=#{yy.namespace}"
```

Comment:
I think instead of this which is setting the namespace for the current context forever, pfab should just specify the desired namespace in all the kubectl commands

Reply:
yeah, I agree.